### ISSUE TITLE
Input: Screen reader does not read error messages automatically

### ISSUE THREAD

**Issue by daneriseb** [seb-oss/green#707]
This was reported as an accessibility issue in our MFE, but needs to be fixed in Green.

"When you focus on any input with an error message the screen reader does not automatically read the error message. We need some way of associating form-info with inputs. "
